adagucserverEC/LayerTypeLiveUpdate/LayerTypeLiveUpdate.cpp

```diff
@@ -105,16 +105,24 @@ int layerTypeLiveUpdateConfigureWMSLayerForGetCapabilities(MetadataLayer *metada
     metadataLayer->layerMetadata.title.copy(metadataLayer->dataSource->cfgLayer->Name[0]->value.c_str());
   }
   CTime timeInstance;
+
+  CT::string timeResolution = "PT10M";
```

Comment:
Maybe mention somewhere in the documentation that this is the default time resolution

Reply:
Wrote a comment in doc/configuration/Layer.md